oat-sa/extension-tao-qtiprint
# Feature/BSA-164/shared stimulus in Booklet

Related to: https://oat-sa.atlassian.net/browse/BSA-164

**Related to: https://github.com/oat-sa/extension-tao-booklet/pull/102**

Added render for `qtiClass: include` (Shared Stimulus)
Added styles for `section.include`

How to test:
1. install `wkhtmltopdf` https://hub.taotesting.com/techdocs/tao-booklet/readme-booklet
2. set `define('DEBUG_MODE', false);` in `config/generis.conf.php`
3. in `config/taoBooklet/wkhtmltopdf.conf.php` set binary to wkhtmltopdf path for example `'binary' => '/usr/local/bin/wkhtmltopdf',`
3. compile bundles for taoQtiPrint and taoBooklet
`cd tao/views/build`
`npx grunt taoqtiprintbundle` 
`npx grunt taobookletbundle` 
4. create shared stimulus
5. create items with shared stimulus
6. create test with these items
7. generate booklet from test
8. open booklet in preview and download pdf
9. Should be: 

- each shared stimulus is included only once in the booklet, 
- a shared stimulus is included in the booklet before the first item referencing it, 
- a shared stimulus is always followed by a page break, 
- a shared stimulus is always preceded by a page break
